### PR TITLE
some change for scaling

### PR DESCRIPTION
**Pull request by tonicbupt**
1. add task api
2. fix backend bug, some daemon has no backend ports

### CODE REVIEW

File: eru/app.py [hunk 1]
```diff
@@ -12,8 +12,9 @@
 from eru.models import db
 
 
-blueprints = ('version', 'sys', 'deploy', 'container', 'app', 'websockets', )
-exts = (db, )
+blueprints = ['version', 'sys', 'deploy', 'container',
+              'app', 'websockets', 'scale', 'task', ]
+exts = [db, ]
```
Comment (CMGS) [inline eru/app.py]: = =这是什么梗

Comment (tonicbupt) [inline eru/app.py]: 我有病...

File: eru/views/scale.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+@bp.route('/<name>/<version>/info')
+@jsonify()
+def touch_version_scale_info(name, version):
+    app = App.get_by_name(name)
+    if not app:
+        raise EruAbortException(code.HTTP_NOT_FOUND, 'App %s not found' % name)
+    v = app.get_version(version)
+    if not v:
+        raise EruAbortException(code.HTTP_NOT_FOUND, 'Version %s not found' % version)
+    cs = v.containers.limit(1).all()
```
Comment (CMGS) [inline eru/views/scale.py]: 给！老！子！改！了！

File: eru/views/scale.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    if not cs:
+        raise EruAbortException(code.HTTP_NOT_FOUND, 'Not deployed')
+    c = cs[0]
```
Comment (CMGS) [inline eru/views/scale.py]: 这里也是

**Pull request merged**